### PR TITLE
Support for modifying the quality and dimensions of the selected image.

### LINKED ISSUE #96: Provide image select callback
### Is your feature request related to a problem? Please describe.

I am trying to change selected image url. The url has some parameters, I have to add 'q' parameter in order to change image quality or fetch compressed image from unsplash

https://unsplash.com/blog/the-5-minute-guide-to-image-quality/

### Describe the solution you'd like

I need a callback receiving selected image data and returning changed block data, which will be given to the editor for rendering. In other words, we can add callback, which will be called after image select action and before giving the block data to the editor
Screenshot 2022-11-19 at 17 29 25

### PR DESCRIPTION
**Pull request by JoseZapata3**
feat: add support for selected image quality parameters

This commit introduces the ability to modify the quality and dimensions of selected images by adding parameters to the image URL in the Unsplash element configuration, in accordance with the Unsplash API.

Closes #96

### CODE REVIEW

File: package-lock.json
Comment (L-Zuluaga) [inline package-lock.json:1]: Is this file required?

File: package.json [hunk 1]
```diff
@@ -55,4 +55,4 @@
       "^.+\\.(css|svg)$": "<rootDir>/test/config/assetsTransform.js"
     }
   }
-}
+}
```
Comment (L-Zuluaga) [inline package.json:58]: is this change required?

File: yarn.lock
Comment (L-Zuluaga) [inline yarn.lock:1]: is this change required?

File: src/unsplashClient.js [hunk 2]
```diff
@@ -56,6 +57,22 @@ export default class UnsplashClient {
     };
   }
 
+  /**
+   * dynamic image resizing
```
Comment (L-Zuluaga) [inline src/unsplashClient.js:61]: What do you think?

```suggestion
   * Applies image resize parameters on the given URL.
```

**Review by L-Zuluaga — CHANGES_REQUESTED** [review]

File: package-lock.json
Comment (JoseZapata3) [reply package-lock.json:1]: Already removed from pull request

File: yarn.lock
Comment (JoseZapata3) [reply yarn.lock:1]: Already removed from pull request

File: src/unsplashClient.js (hunk 2 above)
Comment (JoseZapata3) [reply src/unsplashClient.js]: I think it is a good change, I have already applied the suggestion.

File: package.json (hunk 1 above)
Comment (JoseZapata3) [reply package.json]: Already removed from pull request

**Review by L-Zuluaga — APPROVED** [review]
